website/pages/en/querying/managing-api-keys.mdx

```diff
@@ -2,23 +2,34 @@
 title: Managing your API keys
 ---
 
-Regardless of whether you’re a dapp developer or a subgraph developer, you’ll need to manage your API keys. This is important for you to be able to query subgraphs because API keys make sure the connections between application services are valid and authorized. This includes authenticating the end user and the device using the application.
+## Overview
 
-The "API keys" table lists out existing API keys, which will give you the ability to manage or delete them. For each key, you can see its status, the cost for the current period, the spending limit for the current period, as well as total query numbers. You can click the "three dots" menu to edit a given API key:
+API keys are needed to query subgraphs. They ensure that the connections between application services are valid and authorized, including authenticating the end user and the device using the application.
+
+### Create and Manage API Keys
+
+Go to [Subgraph Studio](https://thegraph.com/studio/) and click the **API Keys** tab to create and manage your API keys for specific subgraphs.
+
+The "API keys" table lists existing API keys and allows you to manage or delete them. For each key, you can see its status, the cost for the current period, the spending limit for the current period, and the total number of queries.
+
+You can click the "three dots" menu to the right of a given API key to:
 
 - Rename API key
 - Regenerate API key
 - Delete API key
 - Manage spending limit: this is an optional monthly spending limit for a given API key, in USD. This limit is per billing period (calendar month).
 
+### API Key Details
+
 You can click on an individual API key to view the Details page:
 
-1. The **Overview** section will allow you to:
+1. Under the **Overview** section, you can:
    - Edit your key name
    - Regenerate API keys
    - View the current usage of the API key with stats:
      - Number of queries
      - Amount of GRT spent
-2. Under **Security**, you’ll be able to opt into security settings depending on the level of control you’d like to have over your API keys. In this section, you can:
+2. Under the **Security** section, you can:
+   - Opt into security settings depending on the level of control you’d like to have
```

Comment:
One of these bullet points is not like the others, as in, this is more of a summary of the following ones instead of a feature in itself. So I would keep it as the introduction, but with your better phrasing:
```suggestion
2. Under the **Security** section, you can opt into security settings depending on the level of control you’d like to have. Specifically, you can:
```

Reply:
I like it!